pkg/dependencydiff_result.go

```diff
@@ -44,40 +44,41 @@ func (ct *ChangeType) IsValid() bool {
 	}
 }
 
-// ScorecardResultsWithError is used for the dependency-diff module to record scorecard results and their errors.
-type ScorecardResultsWithError struct {
-	// ScorecardResults is the scorecard result for the dependency repo.
-	ScorecardResults *ScorecardResult `json:"scorecardResults"`
+// ScorecardResultWithError is used for the dependency-diff module to record the scorecard result
+// and a potential error field if the Scorecard run fails.
+type ScorecardResultWithError struct {
+	// ScorecardResult is the scorecard result for the dependency repo.
+	ScorecardResult *ScorecardResult
 
 	// Error is an error returned when running the scorecard checks. A nil Error indicates the run succeeded.
-	Error error `json:"scorecardRunTimeError"`
+	Error error
 }
 
 // DependencyCheckResult is the dependency structure used in the returned results.
 type DependencyCheckResult struct {
 	// ChangeType indicates whether the dependency is added, updated, or removed.
-	ChangeType *ChangeType `json:"changeType"`
+	ChangeType *ChangeType
```

Comment:
I'm curious why the JSON fields are no longer need

Reply:
We'll have a `JSONDependencyCheckReuslt` in PR #2077 to output JSON, like `ScorecardResult` and `JSONScorecardResultV2`, so IMO, we don't need json tags for `DependencyCheckResult`, is that right?

Reply:
ah, you mean this struct is copied into another one. Got it.